spine-dev-62685282c0b9.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
As of second thought, this file must NOT be present in the codebase.

Instead, it should be stored encrypted. And decrypted before the unit test start.